src/dodal/beamlines/i03.py

```diff
@@ -457,3 +458,31 @@ def baton() -> Baton:
     If this is called when already instantiated in i03, it will return the existing object.
     """
     return Baton(f"{PREFIX.beamline_prefix}-CS-BATON-01:")
+
+
+@device_factory()
+def fluorescence_det_motion() -> FluorescenceDetector:
+    """Get the i03 device for moving the fluorescence detector, instantiate it if it hasn't already been.
+    If this is called when already instantiated in i03, it will return the existing object.
+    """
+    return FluorescenceDetector(PREFIX.beamline_prefix)
+
+
+@device_factory()
+def scintillator() -> Scintillator:
+    """Get the i03 scintillator device, instantiate it if it hasn't already been.
+    If this is called when already instantiated in i03, it will return the existing object.
+    """
+    return Scintillator(
+        PREFIX.beamline_prefix,
+        Reference(aperture_scatterguard()),
+        get_beamline_parameters(),
+    )
+
+
+@device_factory()
+def collimation_table() -> CollimationTable:
+    """Get the i03 device for moving the collimation table, instantiate it if it hasn't already been.
+    If this is called when already instantiated in i03, it will return the existing object.
+    """
+    return CollimationTable(PREFIX.beamline_prefix)
```

Comment:
```suggestion
    return CollimationTable(motion_prefix=f"{PREFIX.beamline_prefix}-MO-TABLE-01", power_supply_prefix=f"{PREFIX.beamline_prefix}-MO-PMAC-02:CRATE2_HEALTHY")
```

Now that we don't have to support device_instantiation and its assumptions of a single prefix and name, these devices could be adjusted to be more reuseable

Other than the PMAC power supply